src/lighteval/models/vllm_model.py

```diff
@@ -77,10 +77,7 @@ def __init__(
         self._add_special_tokens = config.add_special_tokens if config.add_special_tokens is not None else False
         self._tokenizer = self._create_auto_tokenizer(config, env_config)
 
-        if config.max_model_length is not None:
```

Comment:
Using the tokenizer length causes vllm to fail for badly configured models like the Mistral ones